openapi.yaml

```diff
@@ -33,33 +35,33 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/Template'
+              $ref: "#/components/schemas/Template"
```

Comment:
You probably have auto formatter but we use single quotes, please revert changes in this file.